PseudoLocalizer.Core/POProcessor.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Karambolo.PO;
+
+namespace PseudoLocalizer.Core
+{
+    /// <summary>
+    /// Applies transforms to string values in PO resource files.
+    /// </summary>
+    public class POProcessor : Processor
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="POProcessor"/> class.
+        /// </summary>
+        public POProcessor()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="POProcessor"/> class.
+        /// </summary>
+        /// <param name="culture">The output culture.</param>
+        public POProcessor(string culture)
+        {
+            Culture = culture ?? throw new ArgumentNullException(nameof(culture));;
+        }
+
+        /// <summary>
+        /// Gets the culture code associated with the processor.
+        /// </summary>
+        public string Culture { get; }
+
+        /// <inheritdoc />
+        public override void Transform(Stream inputStream, Stream outputStream)
+        {
+            var parser = new POParser(new POParserSettings());
+
+            var result = parser.Parse(inputStream);
+
+            if (result.Success)
+            {
+                var inputCatalog = result.Catalog;
+                var outputCatalog = ProcessCatalog(inputCatalog);
+
+                var generator = new POGenerator(new POGeneratorSettings());
```

Comment:
Same principle here for the generator settings.

Reply:
Done